dragonflybsd/README.md

```diff
@@ -112,6 +112,12 @@ are instructions from downloading sources to kernel installation:
 # ln -s /root/kernels/MYKERNEL
 ```
 
+3. Add 3mdeb repo to remotes
+```
+# git remote add 3mdeb https://github.com/3mdeb/DragonFlyBSD.git
+# git checkout 3mdeb/esrt
+```
+
 3. Build kernel (no need to build toolchain, because we're not cross-compiling)
 
 First time:
```

Comment:
@kotylamichal Add below:
`cd /usr/src`

Reply:
Applied in new PR with a structure similar to Sergii document: https://github.com/3mdeb/fwupd-bsd-documentation/pull/14